conf/evolutions/default/1.sql

```diff
@@ -0,0 +1,9 @@
+# --- !Ups
+
+CREATE TABLE USERS ("Id" INTEGER PRIMARY KEY, "Username" VARCHAR NOT NULL, "Password" VARCHAR NOT NULL);
```

Comment:
I think SQL tables and keys tend to be named in lowercase, to distinguish them from keywords. Also, I think `"quotes"` are for data, and `` `backticks` `` are for column names (though both work I guess).